src/main/resources/db/migration/V1.1_001__message_received_precision.sql

```diff
@@ -0,0 +1,4 @@
+
+-- github issue 8 / add millisecond resolution
+alter table message modify received_date timestamp(3) not null;
```

Comment:
Maybe at some point.  But it does raise the issue that the db migrations are generic, but are actually mysql/mariadb specific.